ruby_audit.gemspec

```diff
@@ -15,8 +15,9 @@ Gem::Specification.new do |spec|
                        'version. It complements bundler-audit, providing ' \
                        'complete coverage for your Ruby stack.'
   spec.homepage      = 'https://github.com/civisanalytics/ruby_audit'
-  spec.license       = 'GPLv3'
+  spec.license       = 'GPL-3.0-or-later'
```

Comment:
Looking around at the license declarations in this repo, I think we also need to include the full GPL 3.0 text in LICENSE, and change the README to say this is licensed under GPL 3.0 or later. That could be a followup PR though.